docs/app/pages/components/components-sections/buttons/floating-action-button-ng1/floating-action-button-ng1.component.html

```diff
@@ -41,13 +41,23 @@ <h4>Floating Action Button - Horizontal (Left)</h4>
 <p>The attributes defining this directive are:</p>
 
 <uxd-api-properties>
-    <tr uxd-api-property name="items" type="array" binding="variable" required="true">
+    <tr uxd-api-property name="items" type="array" binding="variable" [required]="true">
       Array of icons and events, that defines the child buttons shown when the primary button is expanded.
       Each element of the array should contain an <code>icon</code> property which should be the name of an icon from the icon set.
       There should also be an <code>event</code> property which should reference a function that will be called when a button is clicked.
+      <br>
+      Additionally, each item can have a <code>tooltip</code> property which, if defined, will display a tooltip when the button is hovered. 
+      The tooltip position can also be specified by adding a <code>tooltipPlacement</code> property with a value 
+      of <code>top</code>, <code>right</code>, <code>bottom</code> or <code>left</code>.
```

Comment:
Can you list the four available properties either in a bullet list or in another table?
Also, use paragraphs instead of br when needed.